sdk/messaging/azeventgrid/build.go

```diff
@@ -0,0 +1,10 @@
+//go:build go1.18
+// +build go1.18
+
+//go:generate autorest ./autorest.md
+//go:generate gofmt -w .
+
+// (not used for now)
+//nono:generate autorest ./autorest.md --rawjson-as-bytes
```

Comment:
What's the plan for this?

Reply:
I'm going to remove it now that I understand the CloudEvent type better. :)